webpack.config.js

```diff
@@ -180,11 +180,11 @@ module.exports.push({
     ]
   },
   externals: {
-    'popper.js': {
-      root: 'Popper',
-      commonjs2: 'popper.js',
-      commonjs: 'popper.js',
-      amd: 'popper.js'
+    'tippy.js': {
```

Comment:
erg, oh yeah, forgot we use this to keep it external. That make sense? We'd maybe want to manage version anyway?

Reply:
@chuckcarpenter @rwwagner90 Could be good to make a note about this in our [`2.x` planning](https://github.com/shipshapecode/shepherd/projects/2)

Reply:
I'd prefer to be more constrained and bundle a specific version of Tippy. That way, we know exactly what code we're supporting instead of creating an infinite possibility space 😄.

Reply:
@BrianSipple that sounds very logical and works for me. With that in mind, should we remove this from being external here?

Reply:
🤔 I'd rather take it on as a follow-up task.

Reply:
https://github.com/shipshapecode/shepherd/issues/263

Reply:
Follow up is fine, as long as it's a quick follow up 😄